app/src/main/kotlin/com/wire/android/mapper/MessageContentMapper.kt

```diff
@@ -4,17 +4,17 @@ import androidx.annotation.StringRes
 import com.wire.android.R
 import com.wire.android.ui.home.conversations.findUser
 import com.wire.android.ui.home.conversations.model.MessageBody
-import com.wire.android.ui.home.conversations.model.MessageContent
+import com.wire.android.ui.home.conversations.model.MessageContent as UIMessageContent
```

Comment:
Perhaps, just like we've done before with `Entity`, `DTO`, etc. we should come up with a suffix for `UI` stuff.

Maybe `ViewState` or `ViewData` for example? But it's fine for now, maybe a topic for the collective ;D

I've been a bit out of Reloaded recently, so I don't know how much of an issue this is.